src/initPostProcessing.es6.js

```diff
@@ -0,0 +1,14 @@
+import { renderer } from "./init.es6.js";
+import THREE from "three";
+
+var composer = new THREE.EffectComposer( renderer );
+composer.addPass( new THREE.RenderPass( scene, camera ) );
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: ["scene" is not defined.](https://www.codacy.com/app/lagora-franck/proc-edu-ria/file/2577057541/issues/source?bid=3286583&fileBranchId=3300960#l5)